device/tt_device.h

```diff
@@ -55,7 +55,7 @@ struct tt_device_dram_address_params {
     std::uint32_t DRAM_BARRIER_BASE = 0;
 };
 /**
- * @brief Struct encapsulating all L1 Address Map parameters required by UMD.
+* Struct encapsulating all L1 Address Map parameters required by UMD.
```

Comment:
This seems mismatched now, right? All the first stars should be in the same column. I think you need a space on this line, and on the */ line, but please check what is the standardized look of comments for doxygen.

Also in a couple of following comments